package customExcepation;

public class Result {
public static void main(String[] args) throws AgeException
{
	
		
	
	Age a=new Age();
	a.age(11);
	
}
}

